docs/pages/machine-id/guides/jenkins.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Jenkins
+description: Using Teleport Machine ID with Jenkins
+---
+
+Jenkins is a open source automation server that is frequently used to build
+Continuous Integration and Continuous Delivery (CI/CD) pipelines.
+
+In this guide, we will demonstrate how to migrate Jenkins workloads to
+Machine ID with minimal changes to your pipelines.
```

Comment:
Maybe we could elaborate on what "migrating Jenkins workloads to Machine ID" means? I think we're talking about accessing remote hosts as actions within jobs rather than providing some alternative to Jenkins itself.

A few usage examples might be handy too, I think scp/sftp to upload logs / release artifacts / etc will be a common use case.